src/main/resources/db/migration/core/V1.01__ADDRESS.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE address
+(
+    `id`       VARCHAR(50) NOT NULL PRIMARY KEY,
```

Comment:
please remove the `

Reply:
I think it would be better if it stays the same, as column `primary` is a keyword.
So you would need to either changed name of the column or add quotes.

Let me know what you think. I think potentially I could rename the column if that's a problem

Reply:
ok